tutorials/2_sprite_animations_gestures/code/pubspec.yaml

```diff
@@ -0,0 +1,24 @@
+name: basic_sprites_animations
+description: Example code for the the tutorial (Basic Sprite and Animations)
+
+version: 1.0.0+1
+
+publish_to: none
+
+environment:
+  sdk: ">=2.12.0 <3.0.0"
+
+dependencies:
+  flame:
+    path: ../../../packages/flame
+  flutter:
+    sdk: flutter
+
+dev_dependencies:
+  flutter_test:
+    sdk: flutter
+  dart_code_metrics: ^2.4.0
+
+flutter:
+  assets:
+    - assets/images/
```

Comment:
this sadly do not work haha

Reply:
What do not work? Assets are working fine

Reply:
I thought you could not import folders? at least you couldn't, did this change??

Reply:
you can import folders, but you need explicitly to leave the `/` in the end, if this was `assets/images`, then it would not work.

Reply:
TIL! that VASTLY simplify my life haha